build_docs.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+export JRUBY_OPTS="-J-Xmx2g"
+export GRADLE_OPTS="-Xmx2g -Dorg.gradle.daemon=false"
+
+export PATH="$HOME/.rbenv/bin:$PATH"
+eval "$(rbenv init -)"
+rbenv local jruby-9.1.12.0
+
+echo "Cloning 'logstash-docs'"
+git clone --depth 1 git@github.com:elastic/logstash-docs.git -b $branch_specifier
```

Comment:
nitpick: safeguard against `$branch_specifier` and other required environment variables being unset:

~~~ sh
git clone --depth=1 git@github.com:elastic/logstash-docs.git -b "${branch_specifier:?}"
~~~

Reply:
I added two checks at the start of the script that will abort unless `$branch_specifier` and `$GITHUB_TOKEN` are defined.